modules/grizzly/src/main/java/org/glassfish/grizzly/nio/transport/UDPNIOConnection.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2009, 2021 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2009, 2025 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Unless you're an Oracle employee, please change this to:
```suggestion
 * Copyright (c) 2025 Contributors to the Eclipse Foundation
 * Copyright (c) 2009, 2021 Oracle and/or its affiliates. All rights reserved.
```

Reply:
Looks good otherwise.